src/app/course/courses-list/courses-list.component.html

```diff
@@ -1,4 +1,13 @@
+<app-search-course (courseSearched)="courseSearched($event)"></app-search-course>
+<div class="add-course-container">
+    <button class="add-course-btn" (click)="addCourse()"><em class="fa fa-plus"></em> Add Course</button>
+</div>
+
 <ng-container>
-    <app-course-item *ngFor="let course of coursesToBeDisplayed;trackBy: coursesTrackFunction" [course]="course" (actionPerformed)="actionPerformedOnItem($event)"></app-course-item>
+    <app-course-item
+        *ngFor="let course of coursesToBeDisplayed; trackBy: coursesTrackFunction"
+        [course]="course"
+        (actionPerformed)="actionPerformedOnItem($event)"
```

Comment:
It would be better to use more informative naming.
and I think it would be more readable to split actionPerforemed into edit and remove
f.e.: (edit )="onEdit($event)"
as far as it is impossible to read from the template what exactly is going on